.gitignore

```diff
@@ -435,3 +435,4 @@ target/
 .project
 
 .pydevproject
+test/xml/
```

Comment:
Is still need this here?

Reply:
It's not needed, we are going to remove it from the ignore list